src/mbed_tools/build/_internal/cmake_file.py

```diff
@@ -50,6 +50,7 @@ def _render_mbed_config_cmake_template(
     options = list(config.options.values())
     macros = list(config.macros.values())
     supported_c_libs = [x for x in target_build_attributes["supported_c_libs"][toolchain_name.lower()]]
+    supported_application_profiles = [x for x in target_build_attributes["supported_application_profiles"]]
```

Comment:
Why do we need to create this new list? Can't we just pass the `target_build_attributes["supported_application_profiles"]` straight to the `context`, as we have done for the other attributes?